internal/config/defaults.go

```diff
@@ -23,24 +19,38 @@ var (
 
 	// operations holds the supported operations inside mesh
 	operations = map[string]interface{}{
-		InstallKuma: map[string]interface{}{
+		InstallKumav071: map[string]interface{}{
 			"type": "0",
 			"properties": map[string]string{
-				"description": "Install kuma service mesh",
-				"version":     "latest",
+				"description": "Kuma service mesh (0.7.1)",
+				"version":     "0.7.1",
+			},
+		},
+		InstallKumav070: map[string]interface{}{
+			"type": "0",
+			"properties": map[string]string{
+				"description": "Kuma service mesh (0.7.0)",
```

Comment:
Thank you for capitalizing "K". Good to stay "Install" only to be consistent with the rest of the UIs. We can perform a sweeping change here though, to remove "Install" when the UX is redone.